### PR TITLE
fix(input): updates input border color to align with core color

### PR DESCRIPTION
**Pull request by hippee-lee**
Signed-off-by: Matt Hippely <mhippely@vmware.com>

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)
- [x] If applicable, have a visual design approval

## PR Type

What kind of change does this PR introduce?
This change updates the input border color to align with the same value used in core (`hsl(198, 10%, 46%)`).
This improves the color contrast to appropriate level.

### Before change
Screen Shot 2022-02-07 at 5 25 03 PM

### After change
Screen Shot 2022-02-07 at 5 24 14 PM

# Input used in
Screen Shot 2022-02-07 at 5 29 41 PM
 the datagrid string filter:

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] clarity.design website / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?
Current input border colors do not have a high enough color contrast. 

Issue Number: N/A

## What is the new behavior?
Inputs now have an acceptable color contrast. 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: projects/angular/src/wizard/providers/wizard-navigation.service.ts [hunk 1]
```diff
@@ -574,6 +574,8 @@ export class WizardNavigationService implements OnDestroy {
     const pageToGoTo = typeof pageToGoToOrId === 'string' ? myPages.getPageById(pageToGoToOrId) : pageToGoToOrId;
     const currentPage = this.currentPage;
 
+    console.log(pageToGoTo);
```
Comment (Shijir) [inline projects/angular/src/wizard/providers/wizard-navigation.service.ts]: Forgot to remove this?

File: projects/angular/src/forms/styles/_variables.forms.scss [hunk 2]
```diff
@@ -15,7 +15,8 @@ $clr-forms-valid-color: $clr-color-success-700 !default;
 $clr-forms-valid-text-color: $clr-color-success-900 !default;
 $clr-forms-subtext-color: $clr-color-neutral-600 !default;
 $clr-forms-placeholder-color: $clr-global-color-construction-600 !default;
-$clr-forms-border-color: $clr-color-neutral-500 !default;
+// Accessibility fox for vpat-7719; this aligns with core
```
Comment (Shijir) [inline projects/angular/src/forms/styles/_variables.forms.scss:18]: ❤️ A11y 🦊

**Review by Shijir — APPROVED** [review]

**Review by mathisscott — APPROVED** [review]
Remove the console.log and we're gtg

File: projects/angular/src/wizard/providers/wizard-navigation.service.ts (hunk 1 above)
Comment (hippee-lee) [reply projects/angular/src/wizard/providers/wizard-navigation.service.ts]: yep, I wIll remove it.

**Review by mathisscott — APPROVED** [review]

**Pull request merged**